Lib/multiprocessing/context.py

```diff
@@ -259,8 +259,9 @@ def get_all_start_methods(self):
         else:
             if reduction.HAVE_SEND_HANDLE:
                 return ['fork', 'spawn', 'forkserver']
-            else:
-                return ['fork', 'spawn']
+            if sys.platform == 'darwin':
+                return ['spawn', 'fork', 'forkserver']
```

Comment:
This still needs to return a list without 'forkserver' when `HAVE_SEND_HANDLE` is false.

```suggestion
            methods = ['spawn', 'fork'] if sys.platform == 'darwin' else ['fork', 'spawn']
            if reduction.HAVE_SEND_HANDLE:
                methods.append('forkserver')
            return methods
```